### PR TITLE
Update Japanese Localizable.strings

### PR DESCRIPTION
**Pull request by nekolaboratory**
* Swipe actions

### CODE REVIEW

File: IceCubesApp/Resources/Localization/ja.lproj/Localizable.strings [hunk 1]
```diff
@@ -161,21 +161,21 @@
...
+"settings.swipeactions.status.action.bookmark" = "ブックマーク";
+"settings.swipeactions.status.action.boost" = "ブースト";
+"settings.swipeactions.status.action.favorite" = "お気に入り";
+"settings.swipeactions.status.action.none" = "何もしない";
+"settings.swipeactions.status.action.quote" = "引用";
+"settings.swipeactions.status.action.reply" = "返信";
+"settings.swipeactions.status.leading" = "右にスワイプ";
+"settings.swipeactions.status.leading.left" = "左";
+"settings.swipeactions.status.leading.right" = "右";
+"settings.swipeactions.status.trailing" = " 左にスワイプ";
```
Comment (te6-in) [inline IceCubesApp/Resources/Localization/ja.lproj/Localizable.strings:175]: I think the leading space is unnecessary. (The English one needs to be fixed too.)

Comment (nekolaboratory) [reply IceCubesApp/Resources/Localization/ja.lproj/Localizable.strings:175]: Fixing only this language first would be inconsistent, so I'll leave it as is until later when I fix the English version, which is the source of the branch.

Thank you for your advice!

**Comment by nekolaboratory** [comment]
@Dimillian Why did you close my PR?

**Comment by Dimillian** [comment]
Because another translation was merged.

**Comment by nekolaboratory** [comment]
Yes. It was committed later than mine.

And some translations are wrong.

**Pull request closed without merging**